README.md

```diff
@@ -12,7 +12,7 @@ E-Bikes is a sample application that demonstrates how to build applications with
 
 -   [Installing E-Bikes using a scratch org](#installing-e-bikes-using-a-scratch-org)
 
--   [Installing E-Bikes using a Developer Edition Org](#installing-e-bikes-using-a-developer-edition-org)
+-   [Installing E-Bikes using a Developer Edition Org or a Trailhead Playground](#installing-e-bikes-using-a-developer-edition-org-or-a-trailhead-playground)
```

Comment:
Good idea. Needs to be added, especially looking at the upcoming QuickStarts, to every sample app.

Reply:
Exactly, that's the whole reason why I did this update :)